minetest.json

```diff
@@ -0,0 +1,51 @@
+{
+    "Minetest": {
+        "containers": {
+            "minetest": {
+                "image": "linuxserver/minetest",
+                "launch_order": 1,
+                "ports": {
+                    "30000": {
+                        "description": "Minetest port. Suggested default: 30000",
+                        "host_default": 30000,
+                        "label": "Minetest port",
+                        "protocol": "udp",
+                        "ui": false
+                    }
+                },
+                "volumes": {
+                    "/config/.minetest": {
+                        "description": "Choose a Share for minetest configuration. Eg: create a Share called minetest-config for this purpose alone.",
+                        "label": "Config Storage"
+                    }
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter a valid UID to run minetest as. It must have full permissions to the Share mapped in the previous step.",
+                        "label": "UID to run minetest as.",
+                        "index": 1
+                    },
+                    "PGID": {
+                        "description": "Enter a valid GID to use along with the above UID. It(or the above UID) must have full permissions to the Share mapped in the previous step.",
+                        "label": "GID to run minetest as.",
+                        "index": 2
+                    },
+                    "TZ": {
+                        "description": "Enter timezone e.g Europe/London (see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).",
+                        "label": "Timezone to run minetest server in.",
+                        "index": 3
+                    }
+                }
+            }
+        },
+        "description": "An open source voxel game engine clone of the popular Minecraft, see <em>more info</em> for details.</p><p>Based on the docker image by <a href='https://www.linuxserver.io/' target='_blank'>Linuxserver.io</a>: <a href='https://hub.docker.com/r/linuxserver/minetest' target='_blank'>https://hub.docker.com/r/linuxserver/minetest</a>, available for amd64, arm32v7 and arm64v8.</p>",
```

Comment:
```suggestion
        "description": "An open source voxel game engine clone of the popular Minecraft, see <em>more info</em> for details.</p><p>Based on the docker image by <a href='https://www.linuxserver.io/' target='_blank'>Linuxserver.io</a>: <a href='https://hub.docker.com/r/linuxserver/minetest' target='_blank'>https://hub.docker.com/r/linuxserver/minetest</a>, available for amd64 and arm64 architecture.</p>",
```

Reply:
@mikey0000 Sorry about this but we have, in the interim in pr #310 recently standardised on our architecture wording for all existing Rock-ons. I know it seems trivial but we are trying to move, step-by-step, too these elements having their own tag and doing this should help with that longer term goal.
Hence my suggestion here.

Thanks for picking-up this pull request again. Nearly there now I think, judging from a quick read of @FroggyFlox's past comments.